test/contconic.jl

```diff
@@ -103,6 +103,27 @@
                               (MOI.VectorAffineFunction{Float64}, MOI.ExponentialCone)   => [[-1., log(5)-1, 1/5]])
         MOIT.exp3test(optimizer, config)
     end
+    @testset "Conic SDP tests" begin
+        optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> MOIU.mock_optimize!(optimizer, ones(3),
+                              (MOI.VectorOfVariables,             MOI.PositiveSemidefiniteConeTriangle) => [[1, -1, 1]],
+                              (MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64})                 => [2])
+        MOIT.sdp0tvtest(optimizer, config)
+        optimizer.optimize! = (optimizer::MOIU.MockOptimizer) -> MOIU.mock_optimize!(optimizer, ones(3),
+                              (MOI.VectorAffineFunction{Float64}, MOI.PositiveSemidefiniteConeTriangle) => [[1, -1, 1]],
+                              (MOI.ScalarAffineFunction{Float64}, MOI.EqualTo{Float64})                 => [2])
+        MOIT.sdp0tftest(optimizer, config)
+        #MOIT.sdp1tvtest(optimizer, config)
```

Comment:
Why is this commented? Any commented-out code should have an explanation or TODO next to it.